gems/sorbet/test/hidden-method-finder/shims.rb

```diff
@@ -0,0 +1,389 @@
+# the following are included to avoid including these in
```

Comment:
```suggestion
# the following are included to get test output to be identical between OS X and Linux
```

Reply:
(the `to avoid` description doesn't describe what does this achieve)